Makefile

```diff
@@ -71,15 +78,20 @@ check: python-env
 # Corrects spelling errors
 .PHONY: misspell
 misspell:
-	go get github.com/client9/misspell
-	${FIND} -name '*'  -exec misspell -w {} \;
+	go get $(MISSPELL_REPO)
```

Comment:
I kind of worry that # of variables in the Makefile is going to explode. For me the two reasons for variables are:

* Needs to be overwritable by beat x
* Multiple use of the same 

Perhaps we can figure out a better way to "store" the variables.